rope/base/builtins.py

```diff
@@ -587,17 +589,21 @@ def get_returned_object(self, args):
 
 class File(BuiltinClass):
 
-    def __init__(self):
+    def __init__(self, filename=None, mode='r', *args):
```

Comment:
It looks like these extra arguments are never used. Are they needed?

Reply:
Yeah they're needed because of [open](https://docs.python.org/2/library/functions.html#open). For example, if I do:
```
with open("file") as f:
    pass
```
I'll get: `TypeError: __init__() takes 1 positional argument but 2 were given`